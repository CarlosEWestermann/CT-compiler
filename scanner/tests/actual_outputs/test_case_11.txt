2 TK_PR_INT [int]
2 TK_IDENTIFICADOR [a]
2 TK_ESPECIAL [=]
2 TK_LIT_INT [5]
2 TK_ESPECIAL [;]
