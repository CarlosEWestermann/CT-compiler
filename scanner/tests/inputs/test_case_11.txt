int globalvar // Missing semicolon here
(bool flag) >= int! functionwitherror {
    return 0;
}
