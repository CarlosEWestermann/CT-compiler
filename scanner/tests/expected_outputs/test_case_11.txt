2: syntax error, unexpected '(', expecting ';' or ','